.github/ci.yaml

```diff
@@ -0,0 +1,26 @@
+name: Run CI Tests
```

Comment:
Should we extract this to its own PR so v1 can benefit and CI runs on this PR before merging? Not a big deal, but if they're independent pieces of work might as well land them independently.

Reply:
:+1:  Yeah let's split the CI migration out